data/client.kv

```diff
@@ -24,9 +24,14 @@
 <BaseButton>:
     ripple_color: app.theme_cls.primaryColor
     ripple_duration_in_fast: 0.2
-<MDTabsItemBase>:
-    ripple_color: app.theme_cls.primaryColor
-    ripple_duration_in_fast: 0.2
+<MDNavigationItemBase>:
+    on_press: app.screens.switch_screens(self)
```

Comment:
`on_release` is actually what was implemented for `TabbedPanelItem` in <= 0.6.1 and `MDTabsItem` in current main. `on_press` is `pass`.

This will break clients expecting calling `on_release` to keep working the same way.

Calling event handlers directly to programmatically switch views is sketchy in the first place, but it's the closest thing to a common interface we have between Kivy and KivyMD